src/test/java/net/openhft/chronicle/queue/impl/single/StoreAppenderInternalWriteBytesTest.java

```diff
@@ -158,12 +157,12 @@ public void run() {
                                 assertEquals(Long.toHexString(index), Long.toHexString(dtIndex));
                         }
                         prev.clear().append(buffer);
-//                        if (index %17 == 0) {
+//                        if (false && index %17 == 0) {
```

Comment:
Unintentional change here? Regardless, perhaps just removing is the better option, paired with the brace below.